company/uber/EncodeAndDecodeTinyURL.java

```diff
@@ -6,33 +6,18 @@
 //and the tiny URL can be decoded to the original URL.
 
 public class EncodeAndDecodeTinyURL {
-    HashMap<String, String> map = new HashMap<String, String>();
-    String characters = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
-    int count = 1;
-
-    public String getKey() {
-        String key = "";
-        while(count > 0) {
-            count--;
-            key += characters.charAt(count);
-            count /= characters.length();
-        }
-        
-        return key;
-    }
     
+    final Map<String, String> map = new HashMap<>(); 
     // Encodes a URL to a shortened URL.
     public String encode(String longUrl) {
-        String key = getKey();
+        final String key = PREFIX + (map.size() + 1);
```

Comment:
PREFIX is not defined anywhere.

Reply:
thanks for point it out. code has been updated.